server/src/main/java/com/uj/projects/booksplatform/review/repository/ReviewRepository.java

```diff
@@ -0,0 +1,16 @@
+package com.uj.projects.booksplatform.review.repository;
+
+import com.uj.projects.booksplatform.book.entity.Book;
+import com.uj.projects.booksplatform.review.entity.Review;
+import com.uj.projects.booksplatform.user.entity.User;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+import java.util.List;
+
+@Repository
+public interface ReviewRepository extends JpaRepository<Review, Integer> {
+    List<Review> findReviewsByBook(Book book);
+    List<Review> findReviewsByUser(User user);
+    Integer countByBook(Book book);
```

Comment:
countBy(...)

Reply:
I can't do this. These are the default names from JPA